packages/react-percy-webpack/src/entry/writeRenderEntry.js

```diff
@@ -1,12 +1,13 @@
-import fs from 'fs';
 import { GlobalVariables, RootElementId } from './constants';
+import escapePathForWindows from './escapePathForWindows';
+import fs from 'fs';
 
 export default function writeRenderEntry(percyConfig, filePath, resolver = require.resolve) {
   fs.writeFileSync(
     filePath,
     `
     const url = require("url");
-    const renderer = require("${resolver(percyConfig.renderer)}");
+    const renderer = require("${escapePathForWindows(resolver(percyConfig.renderer))}");
```

Comment:
@thomashuston Is this going to substitute \ with / in any files that confusingly have \ in their name, even if not running on windows?

Reply:
While I wouldn't expect to see a file with \ in it's name, it's possible.  We had a guy with a : in a folder name a couple of weeks back.

Reply:
I think we pretty much use the same process as this in our other javascript packages though..